src/Microsoft.Health.Fhir.Core/Features/Conformance/Models/ListedCapabilityStatement.cs

```diff
@@ -0,0 +1,55 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Fhir.Core.Features.Conformance.Models
+{
+    public class ListedCapabilityStatement
+    {
+        internal const string ServerMode = "server";
+
+        public ListedCapabilityStatement()
+        {
+            Status = new DefaultOptionHashSet<string>("draft", StringComparer.Ordinal);
```

Comment:
Hi @brendankowitz, how should the CapabilityStatement.Status be set to "active" if the model does not allow for changing it?